shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/persist/service/impl/DataSourcePersistServiceTest.java

```diff
@@ -61,7 +61,7 @@ public void assertLoad() {
     @SneakyThrows({IOException.class, URISyntaxException.class})
     private String readDataSourceYaml(final String path) {
         return Files.readAllLines(Paths.get(ClassLoader.getSystemResource(path).toURI()))
-                .stream().filter(each -> StringUtils.isNotBlank(each) && !each.startsWith("#")).map(each -> each + System.lineSeparator()).collect(Collectors.joining());
+                .stream().filter(each -> StringUtils.isNotBlank(each) && !each.startsWith("#")).map(each -> each + "\n").collect(Collectors.joining());
```

Comment:
It shouldn't be fixed here. `System.lineSeparator()` does return the line separator, it will return the corresponding line separator according to the current computer system. Or you could try to use the `System.getProperty("line.separator")`.